src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -899,29 +1022,32 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
     @autobind
     @action
     private clusterHeatmapByIndex(index:TrackGroupIndex) {
+
         if (this.oncoprint) {
             this.oncoprint.resetSortableTracksSortDirection();
         }
 
-        const groupEntry = this.molecularProfileIdToHeatmapTracks.entries().find(
-            x=>x[1].trackGroupIndex === index
+        const groupEntry = _.values(this.molecularProfileIdToHeatmapTracks).find(
+            trackGroup=>trackGroup.trackGroupIndex === index
         );
-        if (groupEntry) {
-            this.sortMode = {
-                type:"heatmap",
-                clusteredHeatmapProfile: groupEntry[1].molecularProfileId
-            };
-        }
+
+        //ADAM-TODO: is heapmap correct value for sorby param?  if so, adjust type
```

Comment:
@adamabeshouse i commented this out to avoid ts error (so e2e tests would run again). see note.